README.md

```diff
@@ -183,11 +184,18 @@ After the private hosted zones exist, create the data store stack:
 $ make data-store-stack
 ```
 
-#### Launch the Tile and Application Servers
+#### Launch the Application Servers
 
-Now that the VPC and data store stacks are setup, we can launch the tile and applications server stacks, which will make use of the most recent AMIs available:
+Now that the VPC and data store stacks are setup, we can launch the application server stack, which will make use of the most recent AMIs available:
```

Comment:
Are the tile servers removed here because work on the tiler is in-progress?  Or does application encompass the entire redeployable part of the stack? (e.g application == tiler + django app + monitoring)?